projects/plugins/jetpack/modules/shortcodes/youtube.php

```diff
@@ -369,6 +373,22 @@ function youtube_id( $url ) {
 
 	}
 
+	/**
+	 * Format output for Calypso Reader/Notifications/Comments
+	 */
+	if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
+		require_once WP_CONTENT_DIR . '/lib/display-context.php';
+		$context = Display_Context\get_current_context();
```

Comment:
We could make the following change:

```suggestion
		$context = A8C\Display_Context\get_current_context();
```

and then remove the `use A8C\Display_Context;` from the top of the file.

→ similarly as we did with the previous Jetpack-related shortcode fixes, e.g.: [Spotify fix](https://github.com/Automattic/jetpack/pull/20883/files#diff-750d05c44116995dbb4be76792e3909288e990773ed8a1278c36995336f0786bR55).

Reply:
Great catch, I was following an older version of this function and completely missed that. Thanks! Updated 🤜 🤛

Reply:
🤜🏼 🤛🏼